lib/src/main/java/com/auth0/android/lock/utils/CustomField.java

```diff
@@ -56,22 +58,68 @@
         int TYPE_EMAIL = 3;
     }
 
+    /**
+     * Location to store the field into.
+     */
+    @IntDef({PROFILE_ROOT, USER_METADATA})
+    @Retention(RetentionPolicy.SOURCE)
+    public @interface Storage {
+        /**
+         * Store the field into the user profile root.
+         * The list of attributes that can be added to your root profile is here https://auth0.com/docs/api/authentication#signup.
+         */
+        int PROFILE_ROOT = 0;
+
+        /**
+         * Store the field into the user_metadata object.
+         */
+        int USER_METADATA = 1;
```

Comment:
Shouldn't USER_METADATA be default as that's existing behaviour?

Reply:
It is the default https://github.com/auth0/Lock.Android/pull/543/files#diff-2c5ac7c3a0fefe698d1137e6668dc1d1R99